packages/liotad/README.md

```diff
@@ -1,43 +1,56 @@
 
-##PackageThread
+## PackageThread
 
 Currently, supported commands include package action commands and statistical commands.
 
-###Package action commands
+### Package action commands
+
+* **load** package_name <sha1_checksum> package_name_2 <sha1_checksum_package_2>
 
-* **load** package_name sha1_checksum [package_name sha1_checksum] ...
+Load a package with the specified name and its sha1 checksum, even multiple packages can be loaded in the single command as shown above. For example, linux os user can first use "sha1sum filename" cmd to get checksum, and then load package:
 
-Load a package with the specified name and its sha1 checksum. For example, linux os user can first use "sha1sum filename" cmd to get checksum, and then load package by
-"./liotapkg.sh load filename sha1_checksum".
+```bash
+$ ./liotapkg.sh load filename sha1_checksum
+```
 
 A python file of cal_sha1sum.py is also provided to help you calculate checksum for a file:
-python cal_sha1sum.py file_name (could be relative or absolute file name). For example, under /usr/lib/liota/packages,
-python cal_sha1sum.py iotcc_mqtt.py
+
+```bash
+$ python cal_sha1sum.py file_name
+(could be relative or absolute file name). For example, under /usr/lib/liota/packages,
+$ python cal_sha1sum.py iotcc_mqtt.py
+```
 
 If the specified package provides with a list of dependencies, recursively load all its dependencies. If more than one package names are specified, load them (as well as their dependencies) in a batch and no package will be loaded twice or reloaded.
 
 Liota packages must follow certain formats for package manager to process them correctly. It is up to the package developer to follow the format requirements. Details will be provided in later parts of this document. Please also refer to `packages` and `packages/example` for example packages we have provided.
 
+
 If dependency lists of specified packages and their dependencies contain loops, all packages involved in the loop are not to be loaded.
 
 * **unload** package_name [package_name] ...
 
 Unload a package with the specified name. If the specified package has dependents loaded, recursively unload all its dependents. If more than one package names are specified, unload them (as well as their dependents) in a batch.
 
 To `unregister` an entity while unloading set the following flag in `packages/sampleProp.conf` to `True`:
+
 ```bash
 ShouldUnregisterOnUnload = "True"
 ```
 
-* **reload** package_name sha1_checksum
+* **reload** package_name <sha1_checksum>
 
 Unload a package with the specified name and attempt to reload the same package **using the same file name**. Batch operation is not supported for reloading. If the specified package is not loaded when this command is invoked, the command will fail.
 
-* **update** package_name sha1_checksum [package_name sha1_checksum] ...
+* **update** package_name <sha1_checksum> ...
```

Comment:
Can all the commands be mentioned with multiple packages? It might be good to mention which all can use multiple packages as parameters.

Reply:
Better to mention it in comments rather then showcasing in the command line.

Reply:
What i meant is, can all commands like unload, update, reload have multiple packages as arguments? It will good to mention in comments which all commands can take multiple packages as inputs and which one's not.

Reply:
It has been added in in the comments.